### PR TITLE
Modules Additions, Manuals and JSONs Updates

### PR DESCRIPTION
**Pull request by Rdzanuwu**

### CODE REVIEW

File: HTML/Drive-In Window.html [hunk 1]
```diff
@@ -28,24 +27,23 @@
                 <span class="page-header-section-title">Drive-In Window</span>
             </div>
             <div class="page-content">
-                <img src="img/Component/Drive-In Window.svg" class="diagram">
+                <img src="img/Component/DIWsvg.svg" class="diagram">
```
Comment (sleepin-fang) [inline HTML/Drive-In Window.html:30]: Be sure to look out for reverts.

File: HTML/Rebooting M-OS.html [hunk 2]
```diff
@@ -272,25 +272,25 @@ <h2>Calculating Value A</h2>
...
             <div class="page page-bg-02">
                 <div class="page-header">
                     <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
-                    <span class="page-header-section-title">Rebooting M-OS</span>
+                    <span class="page-header-section-title">Rebooting Mystery OS</span>
                 </div>
                 <div class="page-content">
                     <h2>Calculating Value B</h2>
-                    <p>For this you need to modify your A value. Apply to value A 4 operations based on the square's colors in the order they appear on it. This is your value B.</p>
+                    <p>For this, you need modify your value A. Apply to value A 4 operations based on square's colors in order, they appears on it. This is your value B. </p>
```
Comment (sleepin-fang) [inline HTML/Rebooting M-OS.html:293]: You're trying to revert all grammatical errors.

Comment (Rdzanuwu) [reply HTML/Rebooting M-OS.html:293]: huh, i just updated it with the new one provided in request.

File: HTML/Rebooting M-OS.html [hunk 3]
```diff
@@ -21,24 +21,24 @@
...
                 </div>
                 <div class="page-content">
                     <img src="img/Component/Rebooting M-OS.svg" class="diagram">
-                    <h2>On the Subject of Rebooting M-OS</h2>
+                    <h2>On the Subject of Rebooting Mystery OS</h2>
                     <p class="flavour-text">Make a big reboom!</p>
                     <p>
                         This module use an unknown operation system. We're calling it "M-OS". At the start of each bomb,
                         this OS will display a fatal error message. However, there is just a moving square instead of any text. The square will cycle through 4 of 7 possible colors: red, green, blue, yellow, cyan, magenta and white.
-                        To defuse it, simply reboot M-OS. Unfortunately, the bomb's time greatly affects M-OS and turns it into an <i>advanced version</i>, which is harder to reboot.
```
Comment (sleepin-fang) [inline HTML/Rebooting M-OS.html:33]: Most of the line changes still reverts the grammatical errors.

File: HTML/Rebooting M-OS.html [hunk 4]
```diff
@@ -455,7 +455,7 @@ <h2>Final modifications</h2>
                         </tr>
                         <tr>
                             <td>3</td>
-                            <td>2A - (C % B)</td>
+                            <td>2A - (B % C)</td>
```
Comment (sleepin-fang) [inline HTML/Rebooting M-OS.html:458]: Please clarify with author again about all the changes on this file. Thank you.

Comment (Rdzanuwu) [reply HTML/Rebooting M-OS.html:458]: I confirmed with the author the current manual i have in my pr is correct.

File: HTML/Rebooting M-OS.html (hunk 3 above)
Comment (Rdzanuwu) [reply HTML/Rebooting M-OS.html:33]: The same thing is currently on the repo.

**Pull request merged**